packages/store-ui/src/atoms/Spinner/Spinner.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import Spinner from './Spinner'
+
+describe('Spinner', () => {
+  it('`data-store-spinner` is present', () => {
+    const { getByTestId } = render(<Spinner data-testid="loading" />)
```

Comment:
What do you think about this suggestion? This follow the pattern of other components.
```suggestion
    const { getByTestId } = render(<Spinner data-testid="store-spinner" />)
```